packages/desktop-client/src/components/modals/CreateLocalAccountModal.tsx

```diff
@@ -49,7 +55,12 @@ export function CreateLocalAccountModal({
               event.preventDefault();
 
               const nameError = !name;
-              setNameError(nameError);
+              setNameError({ error: nameError, message: 'Name is required' });
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we can put this in a conditional and return early if the error is encountered. Saves us a _tiny_ amount of compute.

```suggestion
              if (!name) {
                  setNameError({ error: true, message: 'Name is required' });
                  return;
              }
```